gulp/config.js

```diff
@@ -232,20 +232,7 @@ module.exports = (function() {
     build: temp + 'styles/',
     output: cssFile,
     options: {
-      // Only includes the styles if @imported
-      // Remember to then update exclude in wiredepOptions if using @import
-      loadPath: [
-        bower + 'bootstrap-sass-official/assets/stylesheets/',
-        bower + 'font-awesome/scss/',
-        bower + 'patternfly-sass/assets/stylesheets/',
-        // Hack so that font-awesome/variables can be found
-        client + 'assets/sass/'
-      ],
-      style: 'compact',
-      noCache: false,
-      compass: false,
-      bundleExec: true,
-      sourcemap: false,
+      outputStyle: 'compact',
```

Comment:
Agreed on `compressed` - although the difference is negligible (before 412 KB, compact 504 KB, compressed 500 KB)